output.json

```diff
@@ -3999,6 +4037,162 @@
+            "required": true,
+            "description": "The unique identifier of the workflow run.",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A workflow run object.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/paths/~1workflow_runs/post/responses/201/content/application~1json/schema"
+                }
+              }
+            }
+          },
+          "default": {
+            "$ref": "#/paths/~1ping/get/responses/default"
+          }
+        }
+      }
+    },
+    "/workflow_runs/{workflow_run_id}/task": {
+      "get": {
+        "summary": "The tasks of a workflow can be retrieved by calling this endpoint with the unique identifier of the workflow run.",
+        "operationId": "list_tasks",
+        "parameters": [
+          {
+            "name": "workflow_run_id",
+            "in": "path",
+            "required": true,
+            "description": "The unique identifier of the workflow run to which the tasks belongs.",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A subset of task objects that were already started or completed, ordered by the created_at field, in ascending order.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/paths/~1workflow_runs/tasks/get/responses/200/content/application~1json/schema"
+                }
+              }
+            }
+          },
+          "default": {
+            "$ref": "#/paths/~1ping/get/responses/default"
+          }
+        }
+      }
+    },
+    "/workflow_runs/{workflow_run_id}/tasks/{task_id}": {
+      "get": {
+        "summary": "A single task can be retrieved by calling this endpoint with the unique identifier of the task and workflow run.",
+        "operationId": "retrieve_task",
+        "parameters": [
+          {
+            "name": "workflow_run_id",
+            "in": "path",
+            "required": true,
+            "description": "The unique identifier of the workflow run to which the task belongs.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "task_id",
+            "in": "path",
+            "required": true,
+            "description": "The identifier of the task you want to retrieve.",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A task object.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/paths/~1workflow_runs/tasks/get/responses/200/content/application~1json/schema"
```

Comment:
The schema is the same as for the list_tasks, is this correct?

Reply:
The list tasks returns a subset of the Task object. Basically it excludes the "input" and "output" objects, which can be quite big (for example, the output is a payload similar to the one returned by our reports, which is big for reports such as document or watchlist). 

Thus, in order to keep this endpoint response with a reasonable size, we decided to only return "input" and "output" on the retrieve task endpoint.